src/components/ShareToken/ShareToken.js

```diff
@@ -37,19 +38,36 @@ const ShareToken = (props) => {
 
   return (
     <>
-      <h2>Welcome to your smart shopping list!</h2>
-      <p>Join an existing shopping list by entering a three word token:</p>
+      <img src={Logo} alt="avoCart logo" style={{ width: '50%' }} />
+      <Typography
+        variant="h1"
+        style={{ color: 'white', paddingTop: '2rem', paddingBottom: '2rem' }}
+      >
+        Welcome to your smart shopping list!
+      </Typography>
+      <Typography
+        variant="body1"
+        style={{ color: '#F2F3F2' }}
+        className="instructions"
+      >
+        Join an existing shopping list <br />
+        by entering a three word token:
+      </Typography>
       <StyledForm id="shareToken" onSubmit={onSubmitHandler}>
-        <TextField
-          id="shareToken"
-          variant="outlined"
-          label="Token"
-          onChange={onChangeHandler}
-          required
-        />
-        <Button color="primary" size="small" type="submit" variant="outlined">
-          Join existing list
-        </Button>
+        <WhiteBGWrapper>
+          <TextField
+            id="shareToken"
+            variant="outlined"
+            label="Token"
+            onChange={onChangeHandler}
+            required
+          />
```

Comment:
@cafloyd I think make the `label` a `placeholder` instead. When the label floats up, it doesn't really work against the background. You'll have to also add a label for the input, but that could be hidden in some way. 
![image](https://user-images.githubusercontent.com/10378331/120073683-06984280-c05f-11eb-8889-31434aec30e0.png)

Reply:
Very good point, @laurenmbeatty ! I actually already had to artificially add a white background to this element to get the look we were going for, so another solution would be to add a little bit of padding so you can read the label: 
![Screen Cast 2021-05-29 at 1 26 52 PM](https://user-images.githubusercontent.com/58532369/120079465-dc0bb100-c081-11eb-90e4-854173f58e9b.gif)
Not sure which solution is preferable as far as having a cohesive look across the app, so I'll pose the question on slack!

Reply:
Resolved this issue using the second solution based on feedback from the group!